peepingtom/datablocks/simpleblocks/meshblock.py

```diff
@@ -1,13 +1,16 @@
 import numpy as np
 
 from .pointblock import PointBlock
+from ...depictors import MeshDepictor
 
 
 class MeshBlock(PointBlock):
     """
     Data structure for holding n-dimensional mesh data
     """
-    def __init__(self, vertices: np.ndarray, faces: np.ndarray, **kwargs):
+    _depiction_modes = {'default': MeshDepictor}
+
+    def __init__(self, vertices=(), faces=(), **kwargs):
```

Comment:
_depiction_modes defined before init?

Reply:
Yup - that's a class attribute. It's not meant to be an instance-related thing. It's used in the base class to automatically pick the correct class when a mode is given!